### DISCUSSION TITLE
Folders - Sequence by name reverse

### DISCUSSION THREAD

**Discussion by huber75** [photoprism/photoprism#4387] [Q&A]
Hi from Germany,

My pictures are sorted on hard disc eg:

2004-05-23 xxx
2012-11-12 yyy
2024-07-02 zzz

"Originals" view is - of course - identical to the library on hard disc in alphabetical order, eldest folder on top

I am used to view my folders at smartphone starting with the newest folder, hence, in photoprism, I use the "folder" structure "newest first", but, as some folders have been added later they are not sorted all in correct order.
Then I switch to view my folders "by name", unfortunately they are shown in the rankling as they are at the hard disc, eldest first. Solution would ne "by name reverse".

Any chance for me to change it?

Thanks for any reply, Peter

**Comment by graciousgrey** [comment]
Hello :wave: 

It is not yet possible to sort by name reverse, this is planned for the future:
- https://github.com/photoprism/photoprism/issues/683

**Comment by huber75** [comment]
Thanks - seems to be a long-lasting plan...
